### DISCUSSION TITLE
Option to add current date and/or time.

### DISCUSSION THREAD

**Discussion by njsubedi** [outline/outline#3306] [Ideas]
It would be handy to have `/date` or `/time` or `/datetime` commands that insert the current date and/or time as the text. If there's a way to write a custom plugin/integration for the same, I'd submit a PR.

Use case: Writing meeting notes.

**Comment by tommoor** [comment]
This seemed like a fun easy win so I knocked it together: https://github.com/outline/outline/pull/3309

**Reply by tommoor** [reply]
This is now live on app.getoutline.com – the commands work in the title and body, pretty handy!

**Reply by roovyshapiro** [reply]
This works great! 
But its not in the list of "/" options and there's no official Outline documentation so if I didn't read this specific discussion I wouldn't know about it. I'm assuming it will be in the changelog, but in the meantime, its another compelling reason for https://github.com/outline/outline/discussions/2926

**Comment by mzeecedric** [comment]
@tommoor the next level of this feature is to have a date picker to optionally pick a date instead of today. So /date `ENTER` pastes the current date (as is) but also presents a monthly calendar that lets me pick a date. Also nice would be to make sure, that sorting dates in a table work. I'd also prefer to see the date nicely formatted instead of a pure text entry to that if I click on a date entry again the date picker lets me set the wanted date.  
![image](https://github.com/user-attachments/assets/a639eb46-9afb-455b-b12e-cb1d414649f0)

**Reply by tommoor** [reply]
Yea, right now it just plain text - really we should create a semantic date element !

**Reply by yqw1996** [reply]
Looking forward to this feature being launched。
